src/CommunityVoices/Public/js/batch-upload-confirm.js

```diff
@@ -0,0 +1,329 @@
+toRemove = []; // after we upload form content, we want to delete all qutotes/sources at the same time
+
+const fieldsToChange=[["originalquote","originalText"],["editedquotes","text"],["intervieworfileaccessdate","dateRecorded"],["interviewerresearcher",'interviewer'],["subattribution","subAttribution"],["quotationmarks","quotationMarks"]]; // add to later
+// ------- VARIOUS USER DEFINED FUNCTIONS CALLED ON JQUERY EVENTS
+
+function updateFieldName(field,newname) { // change names of fields so that they match expected field names when the quote is uploaded e.g. original quote should become original text
+    field.attr("name",newname);
+}
+
+function createDeletePromise(quote) { // after we upload a quote, we should add it to a list to delete (delete happens all at once).
+    toRemove.push(quote);
+}
+
+function handleDeletePromises() { // delete all quotes after sending request
+    toRemove.forEach(function(element) {
+        element.remove();
+    });
+}
+
+function postData(form) {
+    if(! form.is(':empty')) {
+        data = form.serializeArray();
+        $.ajax({
+          url : $("#actualForm").attr('action'),
+          type: $("#actualForm").attr('method'),
+          data: data,
+          success: function (data) {
+              // save for later
+          },
+          error: function (data) {
+              // save for later
+          }
```

Comment:
Uh oh, was this supposed to be done?

Reply:
@Sammidysam See my comment above. Think I can get rid of this.

Reply:
If we refresh or move to a new page upon success, I don't think we need to do much more in JavaScript.  You could consider using the `error` function here, but I would only say that if you are not sufficiently notifying about an error currently.

Reply:
I removed these.  I worry that might mess it up but when I test this more so I will find out.